lib/vtls/bearssl.c

```diff
@@ -340,8 +365,29 @@ static CURLcode bearssl_connect_step1(struct Curl_easy *data,
     return CURLE_SSL_CONNECT_ERROR;
   }
 
+  if(ca_info_blob) {
+    struct cafile_source source = {
+      .type = CAFILE_SOURCE_BLOB,
+      .data = ca_info_blob->data,
+      .len = ca_info_blob->len,
+    };
+    ret = load_cafile(&source, &backend->anchors, &backend->anchors_len);
+    if(ret != CURLE_OK) {
+      if(verifypeer) {
+        failf(data, "error importing CA certificate blob");
+        return ret;
+      }
+      /* Only warning if no certificate verification is required. */
```

Comment:
did you mean:
```suggestion
      /* Only warn if no certificate verification is required. */
```

Reply:
I have just copied from the same openssl part :)